debug.go

```diff
@@ -0,0 +1,5 @@
+//go:build bledebug
+
+package bluetooth
+
+var _debug = true
```

Comment:
There is a `debug` constant already, defined in adapter.go.
I think it's a good idea to expose it (using a build tag, like you've done here) but I'd prefer that to be for all targets, not just nina.

Reply:
I will do that in a separate PR.

Reply:
Sounds good to me.